opae.spec

```diff
@@ -162,12 +177,13 @@ done
 %{_usr}/src/opae/cmake/*
 %{_usr}/src/opae/opae-libs/cmake/modules/*
 
-%{_libdir}/opae/libboard_a10gx.so*
-%{_libdir}/opae/libboard_n3000.so*
-%{_libdir}/opae/libboard_d5005.so*
-%{_libdir}/opae/libboard_n5010.so*
-%{_libdir}/opae/libfpgad-xfpga.so*
-%{_libdir}/opae/libopae-v.so*
+%{_libdir}/opae/libboard_a10gx.so
+%{_libdir}/opae/libboard_n3000.so
+%{_libdir}/opae/libboard_d5005.so
+%{_libdir}/opae/libboard_n5010.so
+%{_libdir}/opae/libboard_n6010.so
+%{_libdir}/opae/libfpgad-xfpga.so
```

Comment:
libfpgad-xfpga.so is a plugin for the fpgad daemon.  Did you mean to include it here, or were you after libxfpga.so?  If we want the fpgad plugins packaged with opae-devel, then we also need libfpga-vc.so here.

Reply:
libfpga-vc is in -*devel
> rpm -ql packaging/opae/rpm/opae-devel-2.0.1-1.fc33.x86_64.rpm | grep libfp
/usr/lib64/libfpgad-api.so
/usr/lib64/libfpgaperf_counter.so
/usr/lib64/opae/libfpgad-vc.so
/usr/lib64/opae/libfpgad-xfpga.so